src/sbvr-api/permissions.ts

```diff
@@ -925,13 +928,16 @@ const getApiKeyActorIdQuery = _.once(() =>
 		},
 	}),
 );
+const apiActorPermissionError = new PermissionError();
```

Comment:
We use two separate errors to get unique stack traces/names so it's possible to see from which of the two places it came from when debugging